File: backend/internal/avaliation/domain/eliminacoes.go
package domain

type Eliminacoes struct {
	FuncaoIntestinal              string `json:"funcao_intestinal"`
	ViaEvacuacaoFisiologica       bool   `json:"via_evacuacao_fisiologica"`
	ViaEvacuacaoEstomia           bool   `json:"via_evacuacao_estomia"`
	TipoEstomia                   string `json:"tipo_estomia"`
	PlacaEstomia                  string `json:"placa_estomia"`
	BolsaEstomia                  string `json:"bolsa_estomia"`
	Diurese                       string `json:"diurese"`
	NumSndDiurese                 string `json:"num_snd_diurese"`
	UsaFralda                     bool   `json:"usa_fralda"`
	TrocasFraldaDia               string `json:"trocas_fralda_dia"`
	SVA                           bool   `json:"sva"`
	CateterismoIntermitente       bool   `json:"cateterismo_intermitente"`
	VezesCateterismoDia           string `json:"vezes_cateterismo_dia"`
	SVD                           bool   `json:"svd"`
	NumSVD                        string `json:"num_svd"`
	ViasSVD                       string `json:"vias_svd"`
	IrrigacaoSVD                  bool   `json:"irrigacao_svd"`
	DataTrocaSVD                  string `json:"data_troca_svd"`
	VolumeDiureseSVD              string `json:"volume_diurese_svd"`
	Cistostomia                   bool   `json:"cistostomia"`
	NumCistostomia                string `json:"num_cistostomia"`
	ViasCistostomia               string `json:"vias_cistostomia"`
	IrrigacaoCistostomia          bool   `json:"irrigacao_cistostomia"`
	QtdIrrigacaoCistostomia       string `json:"qtd_irrigacao_cistostomia"`
	MedicacaoIrrigacaoCistostomia bool   `json:"medicacao_irrigacao_cistostomia"`
	NomeMedicacaoCistostomia      string `json:"nome_medicacao_cistostomia"`
	VolumeDiureseCistostomia      string `json:"volume_diurese_cistostomia"`
	Preservativo                  bool   `json:"preservativo"`
	VolumeDiuresePreservativo     string `json:"volume_diurese_preservativo"`
}

File: web-forms/src/store/evaluation/eliminacoes.js
import { defineStore } from 'pinia';

export const useEliminacoesStore = defineStore('eliminacoes', {
    persist: true,
    state: () => ({
        eliminacoes: {
            funcao_intestinal: null,
            via_evacuacao_fisiologica: false,
            via_evacuacao_estomia: false,
            tipo_estomia: null,
            placa_estomia: '',
            bolsa_estomia: '',

            diurese: null,
            num_snd_diurese: null,
            usa_fralda: false,
            trocas_fralda_dia: '',
            sva: false,

            cateterismo_intermitente: false,
            vezes_cateterismo_dia: '',

            svd: false,
            num_svd: '',
            vias_svd: null,
            irrigacao_svd: false,
            data_troca_svd: null,
            volume_diurese_svd: null,

            cistostomia: false,
            num_cistostomia: '',
            vias_cistostomia: null,
            irrigacao_cistostomia: false,
            qtd_irrigacao_cistostomia: '',
            medicacao_irrigacao_cistostomia: false,
            nome_medicacao_cistostomia: '',
            volume_diurese_cistostomia: null,

            preservativo: false,
            volume_diurese_preservativo: null
        }
    }),
    actions: {
        reset() {
            this.eliminacoes = {
                funcao_intestinal: null,
                via_evacuacao_fisiologica: false,
                via_evacuacao_estomia: false,
                tipo_estomia: null,
                placa_estomia: '',
                bolsa_estomia: '',
                diurese: null,
                usa_fralda: false,
                trocas_fralda_dia: '',
                sva: false,
                cateterismo_intermitente: false,
                vezes_cateterismo_dia: '',
                svd: false,
                num_svd: '',
                vias_svd: null,
                irrigacao_svd: false,
                data_troca_svd: null,
                volume_diurese_svd: null,
                cistostomia: false,
                num_cistostomia: '',
                vias_cistostomia: null,
                irrigacao_cistostomia: false,
                qtd_irrigacao_cistostomia: '',
                medicacao_irrigacao_cistostomia: false,
                nome_medicacao_cistostomia: '',
                volume_diurese_cistostomia: null,
                preservativo: false,
                volume_diurese_preservativo: null
            };
        }
    }
});